src/xunit.runner.msbuild/xunit.cs

```diff
@@ -200,7 +200,8 @@ public override bool Execute()
             if (NeedsXml)
             {
                 if (Xml != null)
-                    assembliesElement.Save(new FileStream(Xml.GetMetadata("FullPath"), FileMode.OpenOrCreate));
+                    using (var xmlStream = new FileStream(Xml.GetMetadata("FullPath"), FileMode.OpenOrCreate, FileAccess.Write))
```

Comment:
I'm addressing a nit: The `FileStream` constructor defaults to `FileAccess.ReadWrite` and that's not necessary here. But, passing the additional argument isn't mandatory.